### PR TITLE
applications: nrf5340_audio: updated manifest and removed group filter

### PR DESCRIPTION
**Pull request by koffes**
Preparing for making LC3 repo public

Signed-off-by: Kristoffer Rist Skøien <kristoffer.skoien@nordicsemi.no>

### CODE REVIEW

**Review by tejlmand — APPROVED** [review]
lgtm

**Review by carlescufi — APPROVED** [review]

**Review by koffes — COMMENTED** [review]
Great!

File: applications/nrf5340_audio/README.rst [hunk 1]
```diff
@@ -908,14 +870,7 @@ You can build and program the application in one of the following ways:
...
 
-Prerequisites
-=============
-
-Before building the application, make sure to meet the following prerequisites described in the :ref:`nrf53_audio_app_configuration` section:
-
-* :ref:`nrf53_audio_app_configuration_repos`
-
-You might also want to check the :ref:`nRF5340 Audio application known issues <known_issues_nrf5340audio>`.
+You might also want to check the :ref:`nRF5340 Audio application known issues <known_issues_nrf5340audio>` before building and programming the application.
```
Comment (erikrobstad) [inline applications/nrf5340_audio/README.rst:873]:
```suggestion
You might want to check the :ref:`nRF5340 Audio application known issues <known_issues_nrf5340audio>` before building and programming the application.
```

**Review by erikrobstad — APPROVED** [review]

**Review by greg-fer — APPROVED** [review]
Approving my own edits :)

**Review by rick1082 — APPROVED** [review]
Tested with NCS Toolchain manager on Windows with fetching this PR.

File: west.yml [hunk 2]
```diff
@@ -112,7 +112,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: ee0d1c66944f15a47d08883c4826b3b4b4cb317e
+      revision: pull/826/head
```
Comment (rlubos) [inline west.yml:115]: Please update SHA: `9a6637e12575807dc4f6843acc14cc78de4601d5`

**Review by rlubos — APPROVED** [review]

**Comment by rlubos** [comment]
For some reason the CI status was not updated here, but the latest job passed on Jenkins (https://jenkins-ncs.nordicsemi.no/job/latest/job/sdk-nrf/job/PR-8673/8/), merging.

**Pull request merged**